src/Emulator/Peripherals/Peripherals/Video/STM32LTDC.cs

```diff
@@ -121,7 +124,11 @@ protected override void Repaint()
                     }
                 }
 
-                blender.Blend(localLayerBuffer[0], localLayerBuffer[1], ref buffer, backgroundColor, (byte)layer[0].ConstantAlphaConfigurationRegister.Value, (byte)layer[1].ConstantAlphaConfigurationRegister.Value);
+                blender.Blend(localLayerBuffer[0], localLayerBuffer[1], 
+                    ref buffer, 
+                    backgroundColor, 
+                    (byte)layer[0].ConstantAlphaConfigurationRegister.Value, (layer[1].blendingFactor2.Value == BlendingFactor2.MULTIPLY) ? PixelBlendingMode.MULTIPLY : PixelBlendingMode.NO_MODIFICATION,
```

Comment:
is this supposed to be `layer[1].BlendingFactor2` or was it supposed to be `layer[0]`?